src/components/nasjonal-oppgave/__tests__/oppgaveMapping.test.tsx

```diff
@@ -0,0 +1,258 @@
+import { describe, it, expect, beforeEach } from 'vitest'
+import { http, HttpResponse } from 'msw'
+
+import { render, screen } from '../../../utils/testUtils'
+import NasjonalOppgaveView from '../NasjonalOppgaveView'
+import { server } from '../../../mocks/server'
+import { apiUrl } from '../smreg/api'
+import { formatsmregDate, formatsmregDateShorthand } from '../smregDateUtils'
+import {
+    ArbeidsrelatertArsakType,
+    ArbeidsrelatertArsakTypeValues,
+    MedisinskArsakType,
+    MedisinskArsakTypeValues,
+} from '../schema/sykmelding/Periode'
+
+import fullOppgave from './testData/fullOppgave.json'
+import emptyOppgave from './testData/emptyOppgave.json'
+import { mockBehandlerinfo, mockPasientinfo } from './smregTestUtils'
+
+describe('Mapping opppgave fetched from API', async () => {
+    beforeEach(() => {
+        mockPasientinfo()
+        mockBehandlerinfo()
+    })
+
+    it('Should map all fields when "oppgave.papirSmRegistrering" is completely filled out', async () => {
+        server.use(http.get(apiUrl(`/oppgave/${fullOppgave.oppgaveid}`), () => HttpResponse.json(fullOppgave)))
+
+        render(<NasjonalOppgaveView oppgaveId={`${fullOppgave.oppgaveid}`} />, {
+            useRestLink: true,
+        })
+
+        expect(await screen.findByRole('heading', { name: 'Nasjonal papirsykmelding' })).toBeInTheDocument()
+        expect(screen.getByText('Vennligst legg inn opplysningene fra papirsykmeldingen')).toBeInTheDocument()
+
+        // 1 Pasientopplysninger
+        expect(screen.getByLabelText('1.2 Fødselsnummer (11 siffer)')).toHaveDisplayValue(
+            fullOppgave.papirSmRegistering.fnr,
```

Comment:
Oppdaga ein skrivefeil som er tatt med frå smregistrering: 
```suggestion
            fullOppgave.papirSmRegistrering.fnr,
```

Reply:
Den er jo selvfølgelig i backend..